README.md

```diff
@@ -89,6 +89,37 @@ real = "0x0000000000000000000000000000000000000000000000000000000000000000"
 
 Here is a [tool](https://polkadot.subscan.io/tools/ss58_transform) provided by Subscan that helps convert SS58 addresses to public keys. Comment out this field if you don't want to use proxy account.
 
+#### `darwinia_to_ethereum.seed`
+
+Private key in hex of your **Ethereum** account. It's similar to `seed`, but it's for signing on Ethereum network. For example:
+
+```toml
+[darwinia_to_ethereum]
+seed = "0x0000000000000000000000000000000000000000000000000000000000000000"
+```
+
+**For users who want to relay messages from Darwinia to Ethereum, you must request to become a member of the authority set first**:
+
+1. Open [Extrinsics in apps.darwinia.network](https://apps.darwinia.network/#/extrinsics)
+2. If you're not using a proxy account or you have the permission to sign extrinsics using the "real" account, switch to the "real" account and submit `ethereumRelayAuthorities.requestAuthority(stake_amount, signer)`:
+   - `stake_amount` is the amount of RINGs to stake.
+   - `signer` is the public key of your **Ethereum** account.
+    Otherwise, you must switch to your proxy account and submit `proxy.proxy(real_account, ethereumRelayAuthorities.requestAuthority(stake_amount, signer))`. The `stake_amount` will be deducted from your "real" account.
+3. Notify council members to submit `ethereumRelayAuthorities.addAuthority(your_account)`.
+
+> How it works: Authority is a role of relayer. Authorities sign Darwinia-to-Ethereum messages. Once the number of signatures reaches the threshold and the message is delivered and verified on the Ethereum network, the cross-chain is finished. Updating the authority set involves 2 times of cross-chain: 1) relay the new authority set from Darwinia to Ethereum; 2) relay from Ethereum to Darwinia to send rewards to `darwinia_to_ethereum.beneficiary` (see below).
```

Comment:
"Authority is a role of relayer"
I feel not good, but I don't know how to describe it better.

Reply:
Authority is not relayer, authority is the validators/nodes in source chain consensus system to resolve Byzantine Problem, and finalize the blocks.

Grandpa authorities are BFT alike authorities, our authority concept come from the similar meaning, is to be used as replacement for grandpa authorites.

@wuminzhe @wi1dcard

Reply:
Relayer(aka. Bridger) are offchain worker clients who help relay the headers and messages between source chain and target chain, they are working in between two chains and need to access rpcs from two chains.

Our `bridger` repo is an implementation of relayer client.

Reply:
@HackFisher Fixed, please review